.github/workflows/ci_tests.yaml

```diff
@@ -120,6 +120,7 @@ jobs:
             python=${{ matrix.python-version }}${{ matrix.optional-packages }}
             gmt=6.5.0
             ghostscript=10.03.1
+            libgdal-jp2openjpeg
```

Comment:
Do we want to add this here, or in the `gmt`/`pygmt` conda-forge feedstock?

Reply:
I think we should add it to the `gmt` feedstock, otherwise gmt users also can't use these tiled remote datasets.

Reply:
Yeah, it'll be nicer to include it directly as a dependency in [gmt-feedstock](https://github.com/conda-forge/gmt-feedstock/blob/main/recipe/meta.yaml), I'll open a PR for that.